students/Sahlberg/Lesson6/UnitTest.py

```diff
@@ -0,0 +1,92 @@
+"""Ian Sahlberg
+Python210
+2/13/2019
+Unit Testing"""
+
+
+import unittest as u
+import Mailroom3 as m
+from Mailroom3 import amount_append
+from Mailroom3 import donate
+from Mailroom3 import print_report
+from Mailroom3 import quit
+
+
+class tests(u.TestCase):
+
+    """
+    def test_donate(self): #Requires test input = amount below (100)
+        test_dict = {'bugs mouse': [32, 5], 'mickey bunny': {678}}
+        amount_donated = 100
+        expected = 100
+        actual = donate()
+        self.assertEqual(expected,actual)
+    """
+
+    def test_amount_append(self):
+        dict = {'Roberto' : []}
```

Comment:
*really* bad practice - you are overwriting `dict` built-in here
notice that github is highlighting it in a different color